packages/mui-base/src/TextareaAutosize/TextareaAutosize.test.tsx

```diff
@@ -80,8 +81,9 @@ describe('<TextareaAutosize />', () => {
         const { container } = render(<TextareaAutosize />);
         const input = container.querySelector<HTMLTextAreaElement>('textarea[aria-hidden=null]')!;
         const shadow = container.querySelector('textarea[aria-hidden=true]')!;
-        expect(input.style).to.have.property('height', '');
-        expect(input.style).to.have.property('overflow', '');
+
+        expect(input.style).to.have.property('height', '0px');
+        expect(input.style).to.have.property('overflow', 'hidden');
```

Comment:
`height` and `overflow` values are changed from empty strings to '0px' and 'hidden' because previously `getComputedStyleStub` was using `object` and setLayout is mutating this object in below line and setting styles to {height:"",overflow:""}, this is not expected. now since `Map` is being used in place of object we came to know about this bug.

Just as reference i'm attaching codesandbox demo (textarea[aria-hidden=null]), which renders just TextAreaAutoSize, On inspecting `textarea[aria-hidden=true],textarea[aria-hidden=null]` elements , you can see height and overflow  are indeed '0px' and 'hidden'.